### PR TITLE
Adding discord links to readme

### PR DESCRIPTION
**Pull request by slashben**

### CODE REVIEW

**Comment by dwertent** [comment]
Why are you merging to master and not to dev?

**Comment by slashben** [comment]
> Why are you merging to master and not to dev?

No code, only readme

File: README.md [hunk 1]
```diff
@@ -26,8 +24,20 @@ If you wish to scan all namespaces in your cluster, remove the `--exclude-namesp
...
+# Being part of the team
+
+We invite you to our team! We are excited about this project and want to return the love we get. 
+
+Want to contribute? Want to discuss something? Have an issue?
+
+* Open a issue, we are trying to respond within 48 hours
+* [Join us](https://discordapp.com/invite/CTcCaBbb) in a discussion on our discord server! 
+
+[<img src="docs/discord-banner.png" width="100" alt="logo" align="center">](https://discordapp.com/invite/CTcCaBbb)
```
Comment (dwertent) [inline README.md:36]: This is the invite link, what is the logo?

File: docs/run-options.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+helm template [NAME] [CHART] [flags] --dry-run | kubescape scan framework nsa -
+```
+
+for example:
+```
+helm template bitnami/mysql --generate-name --dry-run | kubescape scan framework nsa -
+```
+
+### Offline Support <img src="docs/new-feature.svg">
```
Comment (dwertent) [inline docs/run-options.md:71]: Remove "docs/" from path

File: docs/run-options.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+```
+kubescape scan framework nsa --exclude-namespaces kube-system,kube-public --format json --output results.json
+```
+
+* Output in `junit xml` format 
+```
+kubescape scan framework nsa --exclude-namespaces kube-system,kube-public --format junit --output results.xml
+```
+
+* Scan with exceptions, objects with exceptions will be presented as `warning` and not `fail`  <img src="docs/new-feature.svg">
```
Comment (dwertent) [inline docs/run-options.md:54]: Remove "docs/" from path

**Review by dwertent — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (slashben) [reply README.md:36]: Both and both (I want to have the logo on our page, which is also clickable)

Comment (dwertent) [reply README.md:36]: Oh, I dont see you added the image to the docs dir...

**Review by dwertent — APPROVED** [review]

**Pull request merged**